week-2/F-logical-operators/test2.js

```diff
@@ -0,0 +1,12 @@
+function Sum(n){
+    let i=0
+    let sum=0
+    while (i <= n){
+        
+        sum+=i
+        i++
+    }
+    return sum
+}
```

Comment:
Can you think of a way to do this with `reduce`?

while is useful but can introduce infinite loops quite easily, especially if not tested.